sample/app/src/main/java/com/example/activitystarter/fragment/TabbedPlaceholderFragment.java

```diff
@@ -11,8 +10,10 @@
 
 import activitystarter.ActivityStarter;
 import activitystarter.Arg;
+import androidx.fragment.app.Fragment;
 
-public class TabbedPlaceholderFragment extends Fragment {
+public class TabbedPlaceholderFragment extends Fragment
+{
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html